src/web/handlers.rs

```diff
@@ -71,14 +71,15 @@ pub async fn get_tiles(
         form_factor,
         os_family,
     };
-    if let Some(tiles) = state.tiles_cache.read().await.get(&audience_key) {
-        trace!("get_tiles: cache hit: {:?}", audience_key);
-        metrics.incr("tiles_cache.hit");
-        return Ok(HttpResponse::Ok()
-            .content_type("application/json")
-            .body(&tiles.json));
+    if !state.settings.test_mode {
+        if let Some(tiles) = state.tiles_cache.read().await.get(&audience_key) {
```

Comment:
Just making sure I understand: we don't want to hit the cache when running the integration tests? If that's true, do we still want to write to it when the server is in test mode?

Reply:
Having the server read from the cache means that we have to wait until the cache refreshes or is cleared before we can execute the next test. I'm ok with writing to the cache anyway because this way we're not impacting more of the code. 
I'm honestly debating if in the future we want to make reading from the cache a header designated option as well, but that could introduce subtle bugs if we run tests in parallel.